Tests fail: expected dataType of 9 got 29

Tests fail against latest HANA2
```
1) db NUMBERS execute of Statement should return all numbers like `b`:
     Uncaught AssertionError: expected Parameter {
  dataType: 29,
  fraction: 0,
  ioType: 1,
  length: 16,
  mode: 2,
  name: undefined
} to have property dataType of 9 (got 29)
```
9 - VARCHAR1
29 - STRING

--- a/test/acceptance/db.Prepare.js
+++ b/test/acceptance/db.Prepare.js
@@ -39,7 +39,9 @@ describe('db', function () {
               metadata.should.have.length(1);
               var p = metadata[0];
               p.should.have.property('mode', 2);
-              p.should.have.property('dataType', 9);
+              // on HANA SP12 dataType = 9 (VARCHAR1)
+              // on HANA2 dataType = 29 (STRING)
+              p.should.have.property('dataType').which.is.oneOf(9, 29);
               p.should.have.property('ioType', 1);
               callback(null, statement);
             });

